src/linter/ui5Types/SourceFileLinter.ts

```diff
@@ -830,6 +842,16 @@ export default class SourceFileLinter {
 				!VALID_TESTSUITE.test(this.sourceFile.fileName) &&
 				symbolName === "ready" && moduleName === "sap/ui/core/Core") {
 				this.#reportTestStarter(node);
+			} else if (symbolName === "applySettings" &&
+				nodeType.symbol.declarations?.some((declaration) =>
+					this.isUi5ClassDeclaration(declaration, "sap/ui/core/Control"))) {
+				this.#analyzeNewAndApplySettings(node);
+			} else if (symbolName.startsWith("bindProperty") && moduleName === "sap/ui/base/ManagedObject") {
+				this.#analyzeModelDataTypes(node.arguments[1] as ts.ObjectLiteralExpression);
```

Comment:
Please refrain from using `as`. This here skips the check for undefined values, so undefined might get passed, but the function "analyzeModelDataTypes" doesn't expect undefined to be passed, so it misses checks for undefined values, causing a runtime exception.

Reply:
Thanks!